src/main/java/com/szymczak/livr/Rules/CommonRules.java

```diff
@@ -43,4 +44,32 @@
         return "";
     };
 
+    public static Function<List<Object>, Function<FunctionKeeper, Object>> list_length = objects -> {
+        Object object = objects.get(0);
+        if (object instanceof JSONArray) {
+            JSONArray array = (JSONArray) object;
+            Long minLength = Long.valueOf(array.get(0).toString());
+            Long maxLength = Long.valueOf(array.get(1).toString());
+            return checkArrayLength(minLength, maxLength);
+        } else {
+            final Long length = Long.valueOf(object.toString());
+            return checkArrayLength(length, length);
+        }
+    };
+
+    private static Function<FunctionKeeper, Object> checkArrayLength(Long minLength, Long maxLength) {
+        return (FunctionKeeper wrapper) -> {
+            Object value = wrapper.getValue();
+            if (value == null || (value.toString()).equals("")) return "";
```

Comment:
Is empty list valid? I would consider it as size 0 list and do the validation against the rule. What do you think?

Reply:
Yes I think this would be valid if we wanted to check that the list is empty.

Reply:
Got you. This case handles the object being empty or null 👍